### PR TITLE
Bug fix when warmup is enabled and self.baseline is None.

### LINKED ISSUE #142: About ENAS_Scheduler
While using ENAS_Scheduler
Maybe：
```
tbar.set_description('avg reward: {}'.format(self.baseline))
```
is better than 

```
tbar.set_description('avg reward: {:.2f}'.format(self.baseline))
```
in run in enas_scheduler.py, because when self.baseline=None, it will report error

### PR DESCRIPTION
**Pull request by xingzhaolee**
Fixes #142.

### CODE REVIEW

**Comment by szha** [comment]
Job PR-417-1 is done. 
Docs are uploaded to http://autogluon-staging.s3-website-us-west-2.amazonaws.com/PR-417/1/index.html

**Comment by xingzhaolee** [comment]
@szha when will it be in master?

**Comment by Innixma** [comment]
Hi @xingzhaolee,

Thanks for your contribution! To avoid confusion, szha's message is simply an automated message whenever our CI finishes. We still need to review the code before merging to master.

@zhanghang1989 I have requested review from you since this is an ENAS change. Feel free to request review from @zhreshold if necessary.

File: autogluon/contrib/enas/enas_scheduler.py [hunk 1]
```diff
@@ -126,12 +126,15 @@ def run(self):
...
+                if self.baseline:
+                    tbar.set_description('avg reward: {:.2f}'.format(self.baseline))
                 idx += 1
             self.validation()
             self.save()
-            tq.set_description('epoch {}, val_acc: {:.2f}, avg reward: {:.2f}' \
-                        .format(epoch, self.val_acc, self.baseline))
+            msg = 'epoch {}, val_acc: {:.2f}'.format(epoch, self.val_acc)
+            if self.baseline:
+                msg += ', avg reward: {:.2f}'.format(self.baseline)
```
Comment (zhreshold) [inline autogluon/contrib/enas/enas_scheduler.py:136]: have you tried this, will it becomes too long to fit in one line for progress bar?

Comment (xingzhaolee) [reply autogluon/contrib/enas/enas_scheduler.py:136]: it's able to fit in one line as long as the terminal window isn't too small

**Comment by Innixma** [comment]
@xingzhaolee Thanks for your contribution, it is much appreciated!

**Pull request merged**